venues/OpenReview.net/Support/deployProcess.py

```diff
@@ -35,51 +35,223 @@ def process(client, note, invitation):
 Thanks!
 
 OpenReview Team
-            '''.format(noteId = forum.id, conference_id = conference.get_id(), program_chairs_id = conference.get_program_chairs_id())
+            '''.format(baseurl = client.baseurl, noteId = forum.id, conference_id = conference.get_id(), program_chairs_id = conference.get_program_chairs_id())
         }
     )
     client.post_note(comment_note)
 
     readers = [conference.get_program_chairs_id(), 'OpenReview.net/Support']
-    revision_invitation = client.get_invitation(id= 'OpenReview.net/Support/-/Request' + str(forum.number) + '/Revision')
-    revision_invitation.reply['readers'] = {
-        'values':  readers
-    }
-    revision_invitation.invitees = readers
-    client.post_invitation(revision_invitation)
+    readers.extend(forum.signatures[:])
 
     forum.writers = ['OpenReview.net']
+    forum.readers = readers
     client.post_note(forum)
 
-    reviewer_recruitment_invitation = client.post_invitation(openreview.Invitation(
-        id = 'OpenReview.net/Support/-/Request' + str(forum.number) + '/Reviewer_Recruitment',
-        super = 'OpenReview.net/Support/-/Reviewer_Recruitment',
+    revision_invitation = client.post_invitation(openreview.Invitation(
+        id = 'OpenReview.net/Support/-/Request' + str(forum.number) + '/Revision',
+        super = 'OpenReview.net/Support/-/Revision',
         invitees = readers,
         reply = {
             'forum': forum.id,
-            'replyto': forum.id,
+            'referent': forum.id,
             'readers' : {
                 'description': 'The users who will be allowed to read the above content.',
                 'values' : readers
             }
         },
-        signatures = ['OpenReview.net/Support']
+        signatures = [conference.get_program_chairs_id()]
     ))
 
+    recruitment_email_subject = '[{Abbreviated_Venue_Name}] Invitation to serve as {invitee_role}'.format(Abbreviated_Venue_Name = conference.get_short_name())
+    recruitment_email_body = '''Dear {name},
+
+You have been nominated by the program chair committee of {Abbreviated_Venue_Name} to serve as {invitee_role}. As a respected researcher in the area, we hope you will accept and help us make {Abbreviated_Venue_Name} a success.
+
+You are also welcome to submit papers, so please also consider submitting to {Abbreviated_Venue_Name}.
+
+We will be using OpenReview.net and a reviewing process that we hope will be engaging and inclusive of the whole community.
+
+To ACCEPT the invitation, please click on the following link:
+
+{accept_url}
+
+To DECLINE the invitation, please click on the following link:
+
+{decline_url}
+
+Please answer within 10 days.
+
+If you accept, please make sure that your OpenReview account is updated and lists all the emails you are using.  Visit http://openreview.net/profile after logging in.
+
+If you have any questions, please contact us at info@openreview.net.
```

Comment:
It says "please contact us", but is signed by the PCs.  
"If you have any questions, please contact info@openreview.net."

Reply:
Changed. Thanks.